integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/GroupFhirResourceProviderIntegrationTest.java

```diff
@@ -45,8 +45,6 @@ public class GroupFhirResourceProviderIntegrationTest extends BaseFhirR4Integrat
 	
 	private static final String COHORT_DATA_XML = "org/openmrs/module/fhir2/api/dao/impl/FhirCohortDaoImplTest_initial_data.xml";
 	
-	private static final String PATIENT_DATA_XML = "org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImplTest_initial_data.xml";
```

Comment:
This integration test was actually failing.  Not sure why this was never caught otherwise.  But the tests are wrong - both the COHORT_DATA_XML and PATIENT_DATA_XML were contributing cohorts, but the tests were based only on those in COHORT_DATA_XML.  I limited this to just the one file, and copied stuff from PATIENT_DATA_XML over that was needed.